src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -117,6 +132,7 @@ export const Icons: Story = args => {
 };
 Icons.args = {
   ...Basic.args,
+  placeholder: 'Choose a rating...',
```

Comment:
Could this be "how much do you like tea?" or something?

Reply:
Makes sense, though it could be redundant with the label text:

<img width="272" alt="Screen Shot 2021-08-11 at 7 58 16 PM" src="https://user-images.githubusercontent.com/10625031/129122282-cc314354-6165-4438-90b2-f34115268bbc.png">

Reply:
Fair!